backend/src/nodes/image_util_nodes.py

```diff
@@ -24,26 +24,10 @@ def __init__(self):
         """Constructor"""
         super().__init__()
         self.description = """Blends overlay image onto base image using 
-            specified mode and opacities."""
+            specified mode."""
         self.inputs = [
             ImageInput("Base Layer"),
-            SliderInput(
```

Comment:
Is it really a good idea to remove them? I understand you can achieve the same effect by using the new opacity node, but I still think it's usefull to have them here.

Also, the opacity sliders make it very clear that an RGBA (well, BGRA actually) is returned. Without the sliders, users might expect that e.g. the diff of 2 1-channel images to be a 1-channel image.

Reply:
I've actually toyed with the idea that it should output an image with c = max(base, overlay). I wasn't sure whether it was actually worth it, but I might do that, at least if the node doesn't have its own sliders. As for whether blend should have its own opacity sliders, I think that might warrant some discussion on discord.